### PR TITLE
Adding cards for NMSSM X to YH. Y and H both to WW

### PR DESCRIPTION
**Pull request by OzAmram**
This is the 4+4 prong topology for the CASE analysis, adapted from X to YH, Y->bb cards. Scan of X, Y and H masses

cc @wpmccormack @thaarres

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/param_card.dat [hunk 1]
```diff
@@ -0,0 +1,565 @@
+######################################################################
+## PARAM_CARD AUTOMATICALY GENERATED BY MG5 FOLLOWING UFO MODEL   ####
+######################################################################
+##                                                                  ##
+##  Width set on Auto will be computed following the information    ##
+##        present in the decay.py files of the model.               ##
+##        See  arXiv:1402.1178 for more details.                    ##
+##                                                                  ##
+######################################################################
+
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/param_card.dat:10]: do you really need to param_card on top of customize card. Usually one should pick the default and just add the customization to customize card.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set gauge unitary
+set complex_mass_scheme False
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/proc_card.dat:19]: please remove everything. all default and import model sm will have no effect in your case.

**Comment by agrohsje** [comment]
Note to myself: model is 4FS per default. So settings ok.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/param_card.dat (hunk 1 above)
Comment (OzAmram) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/param_card.dat]: Ok I have removed it

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/proc_card.dat (hunk 2 above)
Comment (OzAmram) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/NMSSM_XYH_WWWW/Template/proc_card.dat]: Ok, removed

**Comment by OzAmram** [comment]
Ok I just pushed a commit with the suggested edits

**Comment by sihyunjeon** [comment]
looks fine to me @agrohsje

**Comment by sihyunjeon** [comment]
@agrohsje confirmation needed :)

**Comment by agrohsje** [comment]
Dear @OzAmram . Final question before merging: You are using external cross section calculations? So setting the width to 1 makes no difference for you. It is just about having it narrow?

**Comment by OzAmram** [comment]
@agrohsje Yes that is correct

**Pull request merged**